### PR TITLE
public pages: fixes

### LINKED ISSUE #4724: Public map enhancements
I really like new style for public map, but here it is some small issues under iOS 8.4, iPhone 5S:

image
image
image
image

### PR DESCRIPTION
**Pull request by matallo**
close #4724

- no right extra padding (footer)
- restore links color and icon size in dashboard settings

screen shot 2015-07-29 at 11 04 46

- non break `DatasetsList-itemSecondaryInfo` (rows abbr)

screen shot 2015-07-29 at 12 45 23

- add separator in right navmenu
- darker grey on `.Navmenu-rarrow`
- hide likes indicator count on mobile
- same line username with gradient overflow

screen shot 2015-07-29 at 12 44 03
screen shot 2015-07-29 at 12 44 34

CR @CartoDB/frontend

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5462/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5470/))

File: app/views/admin/visualizations/public_map.html.erb [hunk 1]
```diff
@@ -170,7 +172,7 @@
                     <div class="DatasetsList-itemInfo">
                       <div class="DatasetsList-itemPrimaryInfo">
                         <h3 class="DatasetsList-itemTitle DefaultTitle u-ellipsLongText">
-                          <a href="<%= CartoDB.url(self, 'public_table', {id: vis.table.name}, @user_domain.nil? ? nil : @related_tables_owners[vis.table.user_id]) %>" title="<%= vis.table.name %>" class="DefaultTitle-link u-ellipsLongText"><%= vis.table.name %></a>
+                          <a href="<%= CartoDB.url(self, 'public_table', {id: vis.table.name}, @user_domain.nil? ? nil : @related_tables_owners[vis.table.user_id]) %>" title="<%= vis.table.name %>" class="DefaultTitle-link u-ellipsLongText">table_50m_rivers_lake_centerlines_with_scale_r</a>
```
Comment (javierarce) [inline app/views/admin/visualizations/public_map.html.erb]: I don't get this change, sorry. Why is this name hardcoded here?

File: app/views/admin/visualizations/public_map.html.erb [hunk 2]
```diff
@@ -192,13 +194,9 @@
                         <% if (vis.table.service.rows_counted) %>
                           <span class="RowsIndicator u-hideOnTablet">
                             <i class="iconFont iconFont-Rows RowsIndicator-icon"></i>
-                            <%= number_with_delimiter(vis.table.service.rows_counted) %> row<%= vis.table.service.rows_counted.size == 1 ? "" : "s" %>
+                            <%= "#{number_to_human(1611611000, format: "%n%u", units: { unit: "", thousand: "K", million: "M" }, delimiter: ",")} row#{vis.table.service.rows_counted.size == 1 ? "" : "s"}" %>
```
Comment (javierarce) [inline app/views/admin/visualizations/public_map.html.erb]: Same thing here as in the previous comment with the hardcoded number.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5472/))

**Comment by javierarce** [comment]
:+1: :balloon:

**Comment by matallo** [comment]
totally my bad :( I skipped these while testing, shouldn't have to, thanks for the thorough review!

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5475/))

**Comment by xavijam** [comment]
I've just checked this pull request, please compare [my issue(s)](https://github.com/CartoDB/cartodb/issues/4781) with what you have fixed @matallo.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5490/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5491/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5495/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5497/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5500/))

**Comment by javierarce** [comment]
:+1: to the new changes.

**Pull request merged**